lib/spark_api/models/search_template/quick_search.rb

```diff
@@ -5,7 +5,12 @@ class QuickSearch < Base
       include Concerns::Savable,
               Concerns::Destroyable
 
-      self.element_name="searchtemplates/quicksearches"
+      self.element_name = "searchtemplates/quicksearches"
+
+      def fields(args = {})
+        arguments = {:_expand => "Fields"}.merge(args)
```

Comment:
@bhornseth Doesn't line 12 check to see if @fields already exists before doing the request? I coped this format from the Contacts model, but maybe I missed something.